test/secrets_test.rb

```diff
@@ -95,6 +96,13 @@ def response_body(body)
       File.read("SECRET").must_equal("foo")
     end
 
+    it 'logs' do
+      SecretsClient.any_instance.expects(:log).with('secrets found: SECRET,this/is/very/hidden')
```

Comment:
FYI capture can make this read nice ... but won't have a backtrace when it fails ... 
```Ruby
called = []
SecretsClient.any_instance.stubs(:log).capture(called)
called.must_equal ['xxxx']
```